src/modules/userProfile/components/about/useAbout.tsx

```diff
@@ -1,12 +1,28 @@
 import { useState } from 'react';
 import { useSelector } from 'react-redux';
+import { useLoaderData } from 'react-router-dom';
+import { PREFERENCES_CULTURE_QUESTIONS } from 'src/constants/PREFERENCES';
+import { Preference } from 'src/core/api';
 import { verifyAction } from 'src/modules/refer/referUtils';
 import { RootState } from 'src/store';
 
 export const useAbout = () => {
   const identityType = useSelector<RootState, 'users' | 'organizations'>(state => {
     return state.profile.type;
   });
+  const { preferences } = useLoaderData() as { preferences: Preference[] };
```

Comment:
This component is also used for Umaya profile, while we don't get `preferences` in the loader function of Umaya profile. So it causes error while routing to Umaya profile